src/mmw/js/src/core/utils.js

```diff
@@ -31,6 +31,17 @@ var utils = {
         return params;
     },
 
+    /**
+     * Returns the value of a query param from the URL if it exists or null:
+     * https://stackoverflow.com/questions/901115
+     */
+    getParameterByName: function(name) {
```

Comment:
I like this function a lot better than the `parseQueryString` above. We might consider removing `parseQueryString` entirely and replacing its one usage here: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/user/controllers.js#L23 with something like this:

```javascript
var nextParam = utils.getParameterByName('next');
...
model = new models.ItsiSignUpFormModel({
    ...
    next: nextParam ? nextParam : '/'
});
```